apm-collector/apm-collector-component/client-component/pom.xml

```diff
@@ -101,5 +101,15 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>io.shardingjdbc</groupId>
+            <artifactId>sharding-jdbc-core</artifactId>
+            <version>2.0.3</version>
+        </dependency>
+        <dependency>
+            <groupId>commons-dbcp</groupId>
+            <artifactId>commons-dbcp</artifactId>
+            <version>1.4</version>
+        </dependency>
```

Comment:
dependency and version should be defined in collector root pom.xml 

See here: https://github.com/apache/incubator-skywalking/blob/master/apm-collector/pom.xml#L97-L240